docs/_panels/gerrit.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Gerrit
+description: metrics focused on Gerrit changesets.
+author: Bitergia
+screenshot: sigils/gerrit.png
+created_at: 
+grimoirelab_version: 0.2.37
+layout: panel
+---
+
+This dashboard focuses on metrics derived from Gerrit changesets, which are not created by bots. To this purpose, two 
+filters are set on the top of the dashboard (`Changesets Only` and `NOT Bots`). The goal of the dashboard is to provide 
+insights about the changesets in terms of their status, patchsets and submitters. It also provides information split 
+by repository and organization in order to ease comparisions. The visualizations composing the dashboard are depicted below.
```

Comment:
I think you can rephrase this paragraph as the one before.

Reply:
same as above

Reply:
I tried to address your comment